docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -39,13 +39,18 @@ Configuration
 
 To configure the Kafka connector, create a catalog properties file
 ``etc/catalog/kafka.properties`` with the following contents,
-replacing the properties as appropriate:
+replacing the properties as appropriate.
+
+In some cases, such as when using specialized authentication methods, it is necessary to specify
+additional Kafka client properties in order to access your Kafka cluster. To do so,
+add the ``kafka.config.resources`` property to reference your Kafka config files:
```

Comment:
Please mention that these might be overwritten by explicit properties from `kafka.properties`

Reply:
I added:

> ... Note that configs can be overwritten if defined explicitly in ``kafka.properties``